docs/aerodromes/classc/Nowra.md

```diff
@@ -4,7 +4,146 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
+| Nowra TMA    | | Nowra Approach | 123.500          |NW_APP        |
+| Nowra ADC    | | Nowra Tower  | 118.850         | NW_TWR        |
+| Nowra SMC   ||  Nowra Ground  | 135.850         | NW_GND        |
+| Nowra ACD    | | Nowra Delivery  | 128.350         | NW_DEL       |
+| Nowra ATIS    |   | | 125.65         | YSNW_ATIS       |
+
+## Airspace
+
+### TMA Ownership
+
+The Terminal Control Area (TMA) owns the airspace within R421 up to FL125; optionally R420F extends from FL125 to FL300.
```

Comment:
Move to TCU page. Maybe indicate that while R420F extends to F300, it should only be used when operationally required. Potentially even require an airspace release from BIK, and allow for voiceless coord of all aircraft on published air routes via the NWA TACAN above F125.